stan/math/prim/mat/fun/accumulator.hpp

```diff
@@ -25,16 +26,6 @@ class accumulator {
   std::vector<T> buf_;
 
  public:
-  /**
-   * Construct an accumulator.
-   */
-  accumulator() : buf_() {}
-
-  /**
-   * Destroy an accumulator.
-   */
-  ~accumulator() {}
-
```

Comment:
Just as a note, I assumed this was done by accident? Otherwise this never gets destroyed (unless that's what we want to have happen in prim